lambda/textract_comprehend_medical/env.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Parameters": {
+        "HEALTHLAKE_ENDPOINT": "https://healthlake.us-east-2.amazonaws.com/datastore/83c10afe8566667ac2489c8d989b2c14/r4/",
```

Comment:
Are these something that any external customer can ingest?

Reply:
This is a local test file